app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,56 @@
+<resources>
+    <string name="app_name">Concordia-Campus-Guide</string>
+    <string name="title_activity_maps">Carte</string>
+    <string name="title_activity_routes">RoutesActivity</string>
+    <string name="action_settings">Paramètres</string>
+    <string name="point_of_interest_explore">Explorer</string>
+    <string name="warning_msg">Aucun itinéraire disponible</string>
+    <string name="french">Français</string>
+    <string name="calendar">calendrier google</string>
```

Comment:
Can this be Calendrier Google? (just to stay consistent with the rest)
![image](https://user-images.githubusercontent.com/32105523/78817132-b4024f80-79a0-11ea-9749-764ec30531fb.png)